script.js

```diff
@@ -212,6 +212,10 @@ function storeData() {
 
 form.addEventListener('submit', storeData);
 
+userName.addEventListener('focusout', storeData);
+email.addEventListener('focusout', storeData);
+message.addEventListener('focusout', storeData);
+
 const data = JSON.parse(localStorage.getItem('user'));
 if (data) {
   userName.value = data.UserName;
```

Comment:
- Great implementing the changes, however, there are some issues you need to address.. Please follow the instruction closely and watch the video below.

- [ ] After a user fills all the form inputs, all the data should not be lost on the page reload ***without submitting the form***. Only the `name` and `email` input get prefilled. Please, fix this.
HInt. Reload the page to see if all the data in the inputs and text-area are still prefilled.





https://user-images.githubusercontent.com/59397159/189088798-5178fb51-d114-4eb8-b2f6-653ae5dcddbd.mp4